BabyDragon

```diff
@@ -0,0 +1,34 @@
+#ascii art
+#   _-^--_
+#  |*_ @  )
+#     |   |   |> 
+#|^---|    |_| |>      
+#|------_____--|>
+
```

Comment:
Looks nice, but I dont think we need a picture for it to attack, we'll just use lines 1-7